lib/beaker/answers/version40.rb

```diff
@@ -34,7 +34,7 @@ def generate_answers
       install_update_server = answer_for(@options, :q_install_update_server, 'y')
 
       the_answers.map do |key, value|
-        the_answers[key][:q_update_server_host] = update_server_host
+        the_answers[key][:q_update_server_host] = update_server_host if the_answers[key]
```

Comment:
Following the pattern from the [top of the `pe_utils_spec.rb`](https://github.com/puppetlabs/beaker/blob/master/spec/beaker/dsl/install_utils/pe_utils_spec.rb#L21-L28) file, I had to [create 4.0 hosts](https://github.com/kevpl/beaker/blob/bkr319_installpe_puppetagent/spec/beaker/dsl/install_utils/pe_utils_spec.rb#L418-L424) for my testing.  Doing this results in that value being empty.